command/server/config.go

```diff
@@ -267,6 +292,60 @@ func (b *ServiceRegistration) GoString() string {
 	return fmt.Sprintf("*%#v", *b)
 }
 
+// PluginConfig represents the configuration for a single OCI-based plugin
+type PluginConfig struct {
```

Comment:
I would've expected type to be here, even in map format? 🤔

Reply:
Why do we need ot right now? We just download the binary, the plugin type doesn't matter for this